### PR TITLE
Feat(web-react, web-twig): Introduce Codemods #DS-985

### PR DESCRIPTION
**Pull request by pavelklibani**
### Issue reference

### CODE REVIEW

File: packages/web-react/transforms/__testfixtures__/transformAttributeExample.input.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Button } from '../../src/components/Button';
```
Comment (crishpeen) [inline packages/web-react/transforms/__testfixtures__/transformAttributeExample.input.tsx:2]: The import URL could be shorter, couldn't it?

File: packages/web-react/transforms/transformValueExample.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// Install jscodeshift if you haven't already
+// npm install -g jscodeshift
```
Comment (crishpeen) [inline packages/web-react/transforms/transformValueExample.ts:2]: i think we use yarn everywhere

File: packages/web-react/transforms/transformValueExample.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+// Result:
+// Will replace *-left to *-start etc. Tooltip components imported from '@lmc-eu/spirit-web-react'
+
+const transform = (fileInfo, api) => {
+  const j = api.jscodeshift;
+  const root = j(fileInfo.source);
+
+  // Find import statements for the specific module
+  const importStatements = root.find(j.ImportDeclaration, {
```
Comment (crishpeen) [inline packages/web-react/transforms/transformValueExample.ts:15]: how does this work? The adopters might use:
* @lmc-eu/spirit-web-react
* @lmc-eu/spirit-web-react/components
* @lmc-eu/spirit-web-react/components/Tooltip

Does all of these work?

File: packages/web-twig/transforms/replace-codemod.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+const codemodName = args[1];
+
+// Define your regex pattern and replacement
+switch (codemodName) {
+  case 'placement':
+    var pattern = /top-left/g;
+    var replacement = 'top-start';
+    break;
+  case 'buttonText':
+    var pattern = /buttonLabel/g;
```
Comment (crishpeen) [inline packages/web-twig/transforms/replace-codemod.ts:30]: this matches all buttonLabel everywhere, doesn't it?

File: packages/web-twig/transforms/src/TooltipModern.twig [hunk 5]
```diff
@@ -0,0 +1 @@
+<TooltipModern >tooltip</TooltipModern>
```
Comment (crishpeen) [inline packages/web-twig/transforms/src/TooltipModern.twig:1]: you are missing empty line at the end of the file

File: packages/web-twig/transforms/src/notWorking.twig
Comment (crishpeen) [inline packages/web-twig/transforms/src/notWorking.twig:1]: 🤔

**Review by crishpeen — COMMENTED** [review]
Nice work. Looks promising. I would love to see more docs on how it works and how to use it. Mainly I am a but confused by the twig implementation.

File: packages/web-react/transforms/__testfixtures__/transformAttributeExample.input.tsx (hunk 1 above)
Comment (literat) [reply packages/web-react/transforms/__testfixtures__/transformAttributeExample.input.tsx:2]: It could, but this will require setting an alias for that.

Comment (literat) [reply packages/web-react/transforms/__testfixtures__/transformAttributeExample.input.tsx:2]: https://www.typescriptlang.org/tsconfig#paths

File: packages/web-react/transforms/__testfixtures__/transformAttributeExample.input.tsx
Comment (literat) [inline packages/web-react/transforms/__testfixtures__/transformAttributeExample.input.tsx:1]: Why the `__testfixtures__` and not just a `__fixtures__` that are standard for the Jest? Is there any naming conflict or another problem?

**Pull request closed without merging**